README.md

```diff
@@ -260,6 +260,18 @@ DfE::Analytics.config.anonymise_web_request_user_id = false
 
 Anonymisation of `user_id` would be required if the source field in the schema is in `analytics_pii.yml` so that analysts can join the IDs together. If the `user_id` is not in `analytics_pii.yml` but is in `analytics.yml` then `user_id` anonymisation would *not* be required so that the IDs could still be joined together.
 
+### Data Anonymisation Algorithm
```

Comment:
TIL!

Reply:
@stevenleggdfe agree that pseudonymisation would be better terminology. However, historically it's been called anonymisation in the gem and there are methods such as `DfE::Analytics.anonymise` that are used externally and config variables that would need to be updated as well as all the other documentation. For consistency we would have to update all these and this would be part of a bigger change with external dependencies, so I don't want to tackle this just yet.

Strictly speaking, this is just a hashing algorithm, so I would say it's more correct to call this "hashing" rather than "anonymisation", but that's a different story.

Reply:
@stevenleggdfe I decided to action this review comment in the end as this is a sensitive and important area.

The only item I didn't change is the `anonymised_user_agent_and_ip` field in event data for web requests. I thought this item would require a big migration and also this data doesn't identify a user right ? Only a user session ?

I've kept the `DfE:Analytics.anonymise` for backwards compatibility. We can deprecate in future releases. I believe the only team that use this is tv, although I think it's best to send out deprecation warnings and remove in a more release friendly way.

Reply:
Sounds good to me - thank you!